### PR TITLE
[Tests] more tests for Cmd line parameters

### PR DESCRIPTION
**Pull request by dkijania**

### CODE REVIEW

File: vit-servicing-station-tests/src/tests/bootstrap/arguments/out_settings.rs [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        empty_db,
+        server::{dump_settings, load_settings, BootstrapCommandBuilder, ServerSettingsBuilder},
+    },
+};
+use assert_cmd::assert::OutputAssertExt;
+use assert_fs::{fixture::PathChild, TempDir};
+use std::path::PathBuf;
+use vit_servicing_station_lib::server::settings::ServiceSettings;
+
+#[ignore]
```
Comment (danielSanchezQ) [inline vit-servicing-station-tests/src/tests/bootstrap/arguments/out_settings.rs:13]: Is this one failing?

Comment (dkijania) [reply vit-servicing-station-tests/src/tests/bootstrap/arguments/out_settings.rs:13]: I've reported "question" for that: https://github.com/input-output-hk/vit-servicing-station/issues/62

Comment (danielSanchezQ) [reply vit-servicing-station-tests/src/tests/bootstrap/arguments/out_settings.rs:13]: I opened a fix in #63.

Comment (danielSanchezQ) [reply vit-servicing-station-tests/src/tests/bootstrap/arguments/out_settings.rs:13]: @dkijania this shouldn't be ignored anymore right?

**Comment by dkijania** [comment]
oh, yes. my bad!

**Pull request merged**